subprojects/soak/src/integTest/groovy/org/gradle/jvm/toolchain/JavaToolchainDownloadSoakTest.groovy

```diff
@@ -21,14 +21,38 @@ import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 class JavaToolchainDownloadSoakTest extends AbstractIntegrationSpec {
 
     def setup() {
+        settingsFile << """
+            pluginManagement {
+                repositories {
+                    maven {
+                        url 'https://plugins.grdev.net/m2/'
+                    }
+                }
+            }
+
+            plugins {
+                id 'org.gradle.disco-toolchains' version '0.1'
+            }
+            
+            toolchainManagement {
+                jvm {
+                    javaRepositories {
+                        repository('disco') {
+                            resolverClass = org.gradle.disco.DiscoToolchainResolver
+                        }
+                    }
+                }
+            }
```

Comment:
This makes me concerned about this change -- this is _incredibly_ verbose for what used to be a one-line feature, which is likely to put people off of using it. I know I would be less likely to use toolchains if I had to copy around 12 lines to every single project I made. Is there any way that we could have a smaller way to enable this?

Reply:
You are right, we are addressing this with a convenience plugin, published together with the basic foojay plugin, which, when applied, will automatically add the toolchain management part.